addons/mail/static/src/models/attachment_viewer.js

```diff
@@ -179,19 +180,37 @@ registerModel({
         rotate() {
             this.update({ angle: this.angle + 90 });
         },
+        /**
+         * @private
+         */
+        _computeAttachment() {
+            if (this.attachmentList) {
+                return insertAndReplace({
+                    attachmentOwner: replace(this.attachmentList.selectedAttachment),
+                });
+            }
+            return clear();
+        },
+        /**
+         * @private
+         */
+        _computeAttachments() {
+            if (this.attachmentList) {
+                return insertAndReplace(this.attachmentList.viewableAttachments.map(attachment => {
+                    return { attachmentOwner: replace(attachment) };
+                }));
+            }
+            return clear();
+        },
         /**
          * @private
          * @returns {string}
          */
         _computeImageUrl() {
```

Comment:
This compute is not very useful, I think we can remove the field and directly read from the item.